src/validation_workflow/tar.py

```diff
@@ -0,0 +1,27 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+from .download_utils import DownloadUtils
+from .validation import Validation
+
+
+class Tar(Validation, DownloadUtils):
+    @classmethod
+    def get_architectures(cls) -> list:
+        return ["x64", "arm64"]
```

Comment:
I see this has been returned every .py which does not seems right.
If this is the same for every child class this should exist in the parent class.

Also this does not seems to belongs to a classmethod but more like a property.